.github/workflows/ubuntu.yml

```diff
@@ -53,17 +53,16 @@ jobs:
         if: ${{ env.CPLEX_URL != 0 && env.SOPLEX_URL != 0 }}
         run: |
           # We redirect output of wget to hide the secret URLs.
-          wget $CPLEX_URL &> /dev/null
-          export CPLEX_INSTALLER=cplex_studio129.linux-x86-64.bin
-          chmod +x $CPLEX_INSTALLER
-          ./$CPLEX_INSTALLER -DLICENSE_ACCEPTED=TRUE -DUSER_INSTALL_DIR="$(dirname "${DOWNWARD_CPLEX_ROOT}")" -i silent
-          rm $CPLEX_INSTALLER
+          wget -O cplex_installer $CPLEX_URL &> /dev/null
```

Comment:
We should keep the installer files secret. Thus, the installer files have a random prefix. To no leak the prefix the '-O' option is used here.